src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php

```diff
@@ -93,12 +95,17 @@ public function warmUp(string $cacheDir): array
                     continue;
                 }
 
+                $start = microtime(true);
                 foreach ((array) $warmer->warmUp($cacheDir) as $item) {
                     if (is_dir($item) || (str_starts_with($item, \dirname($cacheDir)) && !is_file($item))) {
                         throw new \LogicException(sprintf('"%s::warmUp()" should return a list of files or classes but "%s" is none of them.', $warmer::class, $item));
                     }
                     $preload[] = $item;
                 }
+
+                if ($io?->isDebug()) {
+                    $io->info(sprintf('"%s" completed in %0.2fms.', $warmer::class, 1000 * (microtime(true) - $start)));
```

Comment:
```suggestion
                    $io->info(strtr('"%class%" completed in %time%ms.', [
                        '%class%' => $warmer::class,
                        '%time%' => sprintf('%0.2f', 1000 * (microtime(true) - $start)),
                    ]));
```

This syntax is easier to read, but ultimately it may not be worth it.

Reply:
not worth it yes :) thanks for having a look